Mailbox.Tests/Tests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+     public class Tests
+    {
+        [TestMethod]
+        public void TestPersonEquals()
+        {
+            string lname = "Kozubenko";
+            Person p1 = new Person("Alla", lname);
+            Person p2_same = new Person("Alla", lname);
+            Person p3_difFname = new Person("Sveta", lname);
+            Person p4_diflname = new Person("Alla", "Jacobs");
+
+            Assert.AreEqual(p1.Equals(p2_same), true);
+            Assert.AreEqual(p1.Equals(p3_difFname), false);
+            Assert.AreEqual(p1.Equals(p4_diflname), false);
+        }
+
+        [TestMethod]
+        public void TestSaveAndLoad()
+        {
+            var FileName = "test.json";
```

Comment:
Local variables should be camelCase.
```suggestion
            var fileName = "test.json";
```